src/pages/home/report/ReportActionCompose.js

```diff
@@ -155,6 +156,7 @@ class ReportActionCompose extends React.Component {
 
             this.focus(false);
         });
+        this.setInputPlaceholder();
         this.setMaxLines();
```

Comment:
Computing input placeholder will cause a regression

Reply:
Ahhhhh yes good point. Ok, back to the drawing board.